redbot/core/cog_manager.py

```diff
@@ -54,18 +55,13 @@ def __init__(self, paths: Tuple[str] = ()):
         """
         conf_paths = [Path(p) for p in await self.conf.paths()]
         other_paths = self._paths
-        core_paths = await self.core_paths()
 
-        all_paths = _deduplicate(list(conf_paths) + list(other_paths) + core_paths)
+        all_paths = _deduplicate(list(conf_paths) + list(other_paths) + [self.CORE_PATH])
 
         if self.install_path not in all_paths:
             all_paths.insert(0, await self.install_path())
         return tuple(p.resolve() for p in all_paths if p.is_dir())
 
-    async def core_paths(self) -> List[Path]:
-        core_paths = [Path(p) for p in redbot.cogs.__path__]
-        return core_paths
```

Comment:
There is only ever one core path, and it never changes during runtime - this has been moved to a static class attribute.